releasenotes/notes/feat-support-langchain-0-1-0d8e0ddd6248c4ed.yaml

```
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    langchain:  This introduces support for ``langchain==0.1.0``. Note that this drops tracing support for deprecated
+    operations in ``langchain==0.1.0`` in favor of recommended alternative methods by [langchain](https://python.langchain.com/docs/changelog/core).
```

Markdown and rst links have different formats. Also I cleaned up the release note a little bit
```suggestion
    langchain operations. Please follow langchain upgrade `guide <https://python.langchain.com/docs/changelog/core>`_ to enable full tracing support.
```